copier/tools.py

```diff
@@ -172,3 +173,56 @@ def cleanup(self):
     @staticmethod
     def _robust_cleanup(name):
         shutil.rmtree(name, ignore_errors=False, onerror=handle_remove_readonly)
+
+
+def get_lines(content: bytes) -> StrSeq:
+    # Assume UTF-8 encoding for text files. Since Jinja2 is already doing this
+    # for templates, there  probably isn't much harm in also doing it for all
+    # text files. The worst case should be a failure to show the diffs. An
+    # alternative is to add chardet support.
+    return content.decode("utf-8").split(os.linesep)
+
+
+def is_binary(content: bytes) -> bool:
+    """Return a boolean indicating if the content is a binary file."""
+    return b"\x00" in content or b"\xff" in content
+
+
+def color_print(color: int, msg: str):
+    print(f"{color}{msg}{colorama.Fore.RESET}")
```

Comment:
You have https://python-prompt-toolkit.readthedocs.io/en/master/pages/printing_text.html#html and https://plumbum.readthedocs.io/en/latest/colors.html for printing colorized output. You probably won't need this function.